src/main/java/com/google/sps/servlets/DestinationDataServlet.java

```diff
@@ -92,21 +95,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             .withObscurity(level)
             .build();
 
-    /* TODO: Store the data in datastore rather than locally */
-    destinations.add(destination);
-    /* TODO: redirect back index.html once the data is stored in datastore */
-    response.sendRedirect(REDIRECT_URL);
-  }
+    /* Takes the destination object and turns it into a JSON String to be stored in Datastore */
+    String jsonDestination = GSON.toJson(destination);
 
-  /* Retrieves the most recent destination created by the user, turns it into a JSON formatted string,
-   *and displays the JSON-ified String on /destination-data
-   */
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Gson gson = new Gson();
-    String json = gson.toJson(destinations.get(destinations.size() - 1));
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
+    Entity destinationEntity = new Entity("Destination");
```

Comment:
Since I imagine this will be needed in multiple servlets, define this "Destination" string in Constants:
https://github.com/googleinterns/step183-2020/blob/master/src/main/java/com/google/sps/servlets/Constants.java